ydb/scheme_test.py

```diff
@@ -0,0 +1,30 @@
+from ydb.scheme import (
+    SchemeEntryType,
+    _wrap_scheme_entry,
+    _wrap_list_directory_response,
+)
+from ydb._apis import ydb_scheme
+
+
+def test_wrap_scheme_entry():
+    assert (
```

Comment:
Is is necessary to parentheses and new lines inside asserts?

```python
assert _wrap_scheme_entry(ydb_scheme.Entry(type=1)).type is SchemeEntryType.DIRECTORY
```

Reply:
I guess it's automatic formatting by black library

Reply:
yes, this is black's trick